viadot/sources/vid_club.py

```diff
@@ -47,22 +47,23 @@ def __init__(self, *args, credentials: Dict[str, Any] = None, **kwargs):
 
     def build_query(
         self,
-        source: Literal["jobs", "product", "company", "survey"],
         from_date: str,
         to_date: str,
         api_url: str,
         items_per_page: int,
+        source: Literal["jobs", "product", "company", "survey"] = None,
+        region: str = "all",
```

Comment:
You can also add type as `Literal["bg", "hu", "hr", "pl", "ro", "si", "all"] = "all"`

Reply:
updated